vignettes/admiral.Rmd

```diff
@@ -283,6 +283,6 @@ channel](https://app.slack.com/client/T028PB489D3/C02M8KN8269).
 
 # See also
 
-- [Template scripts](https://github.com/pharmaverse/admiral/tree/main/inst/templates)
+- [Template scripts](https://github.com/pharmaverse/admiral/tree/main/inst/templatesxyz)
```

Comment:
but does flag this one

Reply:
The string matches a valid URL regex, but the URL itself is inaccessible, hence it's flagged